cloud/docker_client.go

```diff
@@ -77,9 +85,68 @@ func (c *dockerClientImpl) Init(apiVersion string) error {
 
 	// Create HTTP client
 	c.httpClient = util.GetHTTPClient()
+
+	// allow connections to Docker daemon with self-signed certificates
+	_, ok := c.httpClient.Transport.(*http.Transport)
```

Comment:
The first return value is the type converted version of this, so I would use that and set it to true